backend/api/src/main/kotlin/io/tolgee/api/v2/controllers/translation/TranslationsController.kt

```diff
@@ -148,6 +148,15 @@ When null, resulting file will be a flat key-value object.
     )
     @RequestParam(value = "structureDelimiter", defaultValue = ".", required = false)
     structureDelimiter: Char?,
+    @Parameter(
+      description =
+      "Enables filtering of returned keys by their tags.\n" +
+        "Only keys with at least one provided tag will be returned.\n" +
+        "Optional, filtering is not applied if not specified.",
+      example = "?filterTag=productionReady&filterTag=nextRelease",
```

Comment:
The example should not look like this, it will used as default value in the swagger UI.

Reply:
Thanks for the catch, fixed.